### ISSUE TITLE
Update results and analysis with hyperparamters tuned on validation set instead of test

### ISSUE THREAD

**Issue by tallamjr** [tallamjr/astronet#26]
It was discovered that previous analysis was optimised linking to the `x_test` and `y_test`, this should have been the validation set.

This was updated with:
```
commit ca18a44d83ac13b9085619570dd7cf0ae04c64e2 (origin/issues/16/nbval, issues/16/nbval)
Author: Tarek Allam <t.allam.jr@gmail.com>
Date:   Wed Oct 14 18:37:24 2020 +0100

    Change objective function to maximise acc on val

    Previous was maximising objective function on test data which was
    causing overfitting

            modified:   astronet/t2/opt/hypertrain.py
```
```diff
diff --git a/astronet/t2/opt/hypertrain.py b/astronet/t2/opt/hypertrain.py
index ff11057..4163e3a 100644
--- a/astronet/t2/opt/hypertrain.py
+++ b/astronet/t2/opt/hypertrain.py
@@ -117,8 +117,7 @@ class Objective(object):
         model.summary(print_fn=logging.info)

         # Evaluate the model accuracy on the validation set.
-        # score = model.evaluate(X_val, y_val, verbose=0)
-        score = model.evaluate(X_test, y_test, verbose=0)
+        score = model.evaluate(X_val, y_val, verbose=0)
         return score[1]
```
But the results in `**/results.json` should have also been updated. 

For this issue, the previous results should be wiped, and new analysis run with the changes brought in from `ca18a44d8`
